src/vm/eetwain.cpp

```diff
@@ -5169,7 +5169,11 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     if (info.ebpFrame)
     {
         _ASSERTE(stackDepth == 0);
+#if defined(_TARGET_X86_) && defined(FEATURE_PAL)
+        taArgBase = GetCallerSp(pContext) - 2 * sizeof(TADDR);
+#else
         taArgBase = *pContext->GetEbpLocation();
```

Comment:
Now this region is not shared between Windows and Linux, and thus it would be better to use ``*pContext->pEbp`` instead of ``*pContext->GetEbpLocation()``.